Azure CLI - small edits

andynava opened the pull request.

mathewc commented on articles/web-sites-create-web-jobs.md:
```diff
@@ -71,7 +71,7 @@ The following file types are accepted:
 	
-> For continuous WebJobs, it is recommended that you enable **Always On** for your web app. The Always On feature, available in Basic and Standard mode, prevents web apps from being unloaded, even if they have been idle for some time. If your web app is always loaded, your continuously running WebJob may run more reliably. 
+> For continuous WebJobs, it is recommended that you enable **Always On** for your web app. The Always On feature, available in Basic, Standard and Premium modes, prevents web apps and WebJobs from being unloaded, even if they have been idle for some time. without **Always On** enabled your continuous WebJob will stop running after 20 minutes (without querying for it's status).
 
```
The "without querying for it's status" isn't clear to me. How about "Without Always On enabled your continuous WebJob can be stopped (along with the site) when the default web apps idle time threshold is reached (about 20 minutes)."

amitapl replied:
It is important to understand that the WebJob idle time is not related to the web app but to the scm site.
Maybe:

*For Continuous WebJobs to run reliably and on all instances, enable **Always On** configuration for the web app otherwise they will stop after scm site idling out*

mathewc replied:
How about "For Continuous WebJobs to run reliably and on all instances, enable the *Always On** configuration setting for the web app otherwise they can stop running when the SCM host site has been idle for too long."

amitapl commented:
Sold!

andynava committed:
Azure CLI - small edits

andynava committed:
Whitelist only your corporate network

Show how to limit connections to only the user's corporate network instead of whitelisting all of the internet

andynava committed:
SQL Commands

andynava committed:
Select-statement does not do an update

andynava committed:
Be more specific about the first step

andynava commented on articles/postgresql/tutorial-design-database-using-azure-cli.md:
```diff
@@ -102,7 +105,7 @@ The result is in JSON format. Make a note of the **administratorLogin** and **fu
 If your client computer has PostgreSQL installed, you can use a local instance of [psql](https://www.postgresql.org/docs/9.6/static/app-psql.html), or the Azure Cloud Console to connect to an Azure PostgreSQL server. Let's now use the psql command-line utility to connect to the Azure Database for PostgreSQL server.
 
-1. Run the following psql command to connect to an Azure Database for PostgreSQL server
+1. Run the following psql command to connect to an Azure Database for PostgreSQL database:
```
In the next paragraph, it says "connects to the default database", so being consistent about where user is connecting to

andynava commented on articles/postgresql/tutorial-design-database-using-azure-cli.md:
````diff
@@ -140,34 +143,35 @@ You can see the newly created table in the list of tables now by typing:
 UPDATE inventory SET quantity = 200 WHERE name = 'banana';
 ```
 
-The row gets updated accordingly when you retrieve data.
````
Seems to imply that retrieving the data is what is updating the rows

GitHubber17 committed:
Update tutorial-design-database-using-azure-cli.md
